src/BenchmarkDotNet/Toolchains/Roslyn/Generator.cs

```diff
@@ -27,6 +27,7 @@ protected override string[] GetArtifactsToCleanup(ArtifactsPaths artifactsPaths)
 
         protected override void GenerateBuildScript(BuildPartition buildPartition, ArtifactsPaths artifactsPaths)
         {
+            //TODO: For nuget refs i think we need to use the MsBuildArgument/Argument to tell msbuild to fetch nuget refs?
```

Comment:
I think that we can just add a validation rule to the RoslynToolchain and say that this feature is not supported there

https://github.com/dotnet/BenchmarkDotNet/blob/a971a435ce6e6ca25d246e5e2cd56c5b2cf4739d/src/BenchmarkDotNet/Toolchains/Roslyn/RoslynToolchain.cs#L23-L44